src/convertToObject.js

```diff
@@ -7,6 +7,47 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. It will decrease code readability. Consider using a more descriptive name, such as `styleObject` or `cssProperties`.